### ISSUE TITLE
Editor toolbar accessibility: add missing shortcuts

### ISSUE THREAD

**Issue by yannisbarlas** [Coko-Foundation/wax-prosemirror#9]
> **Migrated from GitLab**
> **Original Author:** Dione Mentis (`@DioneMentis`)
> **Original Link:** https://gitlab.coko.foundation/wax/wax-prosemirror/-/issues/118

## Context

If a user is operating without a mouse (keyboard only), keyboard shortcuts are needed to use the tools. The immediate need is shortcuts for the toolset used in the [Assessment Builder](https://gitlab.coko.foundation/hhmi/hhmi) however this applies to all wax tools.  

## Proposal

Create a shortcut for each tool

![Screenshot_2024-07-02_at_12.27.55](/uploads/01c5fda6366289946ba0584377060ddb/Screenshot_2024-07-02_at_12.27.55.png)

In the Assessment Builder, from left to right, the shortcuts are: 

* Undo: `Ctrl` + `z`
* Redo: `Ctrl` + `Shift` + `z`
* Save changes: `Ctrl` +`s` 
* Bold: `Ctrl` +`b` 
* Italics: `Ctrl` +`i` 
* Code: **TBC**
* Strikethrough: **TBC**
* Underline: `Ctrl` +`u` 
* Subscript: **TBC**
* Superscript: **TBC**
* Wrap in ordered (numbered) list: `Shift + Ctrl + 9`
* Wrap in unordered (bulleted) list: `Shift + Ctrl + 8` 
* Lift list item: `Ctrl` + `[` 
* Indent list item: Ctrl-] 
* Insert image: **TBC**
* Insert table: **TBC**
* Full screen: **TBC**
